lib.h

```diff
@@ -106,6 +106,19 @@ struct options {
         bool async_connect;
 
         /* tcp_stream */
+#ifdef WITH_TCPDIRECT
+        bool tcpd_validate;
+        bool tcpd_rx_cpy;
+        const char *tcpd_nic_pci_addr;
+        const char *tcpd_gpu_pci_addr;
+        unsigned long long tcpdirect_phys_addr;
```

Comment:
What is this the phys_addr off? Maybe a more descriptive name.

Reply:
a relic from Sami's patch. Deleted